AudiusProject/audius-protocol
# [PAY-3083] Add user socials in DN

### Description

Part 1 of migrating user socials to DN.

- add dn migration
- update user models and schemas
- update entity manager user verification code to handle new format if relevant fields exist
- remove unused file

### How Has This Been Tested?

- deploy to stage dn and make sure migration ran fine and user models are okay
- select that stage dn while using staging dapp to make sure things still work
